portlets/src/main/webapp/vue-app/engagement-center/components/programs/ProgramDetail.vue

```diff
@@ -202,20 +202,15 @@ export default {
       return [
         {text: this.$t('programs.details.rules.action'), align: 'start', width: '70%'},
         {text: '', align: 'center', width: '15%'},
-        {text: this.$t('programs.details.rules.points'), align: 'center', width: '15%', enabled: !this.isMobile}].filter(filter => filter.enabled == null || filter.enabled === true);
+        {text: this.$t('programs.details.rules.points'), align: 'center', width: '15%'},
+        {text: this.$t('gamification.overview.label.participants'), align: 'center', width: '25%'}];
     },
     displayFooter() {
       return this.totalSize > this.options.itemsPerPage;
     },
     canManageRule() {
       return this.isAdministrator || this.program?.userInfo?.canEdit;
-    },
-    height() {
-      return this.isMobile ? 70 : 173;
-    },
-    isMobile() {
-      return this.$vuetify.breakpoint.smAndDown;
-    },
```

Comment:
Please ensure that the work done [here](https://github.com/Meeds-io/gamification/commit/fc38806b842d62c5be3fcc6a93c54f37aea1cc24) is not reverted